src/sbt-test/sbt-release/doge-integration/test

```diff
@@ -0,0 +1,5 @@
+> release
+> assertArtifactExists library 2.10
```

Comment:
Perhaps it's clearer (and easier) to use `$ exists ...` here.

http://www.scala-sbt.org/0.13/docs/Testing-sbt-plugins.html#step+4%3A+write+a+script